tests/utils/dockercli/volumelifecycle.go

```diff
@@ -53,8 +65,45 @@ func KillDocker(ip string) ([]byte, error) {
 	return out, err
 }
 
+// KillVDVSPlugin - kill vDVS plugin. It is restarted automatically
+func KillVDVSPlugin(ip string) ([]byte, error) {
+	log.Printf("Killing vDVS plugin on VM [%s]\n", ip)
+	out, err := ssh.InvokeCommand(ip, dockercli.GetVDVSPlugin)
+	if err != nil {
+		return out, err
+	}
+
+	pluginID := strings.Fields(misc.FormatOutput(out))[0]
+	out, err = ssh.InvokeCommand(ip, dockercli.KillVDVSPlugin+pluginID)
+
+	time.Sleep(2 * time.Second)
+
+	out, err = ssh.InvokeCommand(ip, dockercli.GetVDVSPlugin)
```

Comment:
following code should be extracted out by creating its dedicated helper ... code flow should be following

test: invoke KillVDVSPlugin util > the util should be responsible for killing plugin
test: verifies whether plugin come back or not <-  util should be provided > if not test should invoke a call to enable plugin (this should be a separate util).